specs/lib/cacheOptimizationSpec.js

```diff
@@ -59,4 +59,24 @@ describe('cacheOptimization', function() {
                 expect(positions).toEqual(unpackedOptimizedVertices);
             }), done).toResolve();
     });
+
+    it('doesn\'t run if a primitive has no indices', function() {
+        var gltf = {
+            meshes : {
+                mesh : {
+                    primitives : [
+                        {
+                            attributes : {
+                                POSITION : 'notImportant',
+                                NORMAL : 'notImportantEither'
+                            }
+                        }
+                    ]
+                }
+            }
+        };
+        expect(function() {
+            cacheOptimization(gltf);
+        }).not.toThrowRuntimeError();
```

Comment:
I just checked and this doesn't fail without the changes, so you're right, we need a better test here.